uber/templates/hotel/no_shows.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base-admin.html" %}
+{% block title %}No Show Staffers{% endblock %}
+{% block content %}
+
+<h2> {{ staffers|length }} Staffers With Hotel Space Who Have Not Checked In </h2>
+
+<table class="list">
+<tr class="header">
+    <td> Staffer Name </td>
+    <td> Departments </td>
+    <td> Room Nights </td>
+</tr>
+{% for attendee in staffers %}
+    <tr>
```

Comment:
BTW, there's a custom filter for this now, so you could just say

```
<td> {{ attendee|form_link }} </td>
```

and it would be the same as what you did here.  Most places haven't been refactored to use the new ``|form_link`` filter yet, though.